content/blog/2018/07/2018-07-25-contributor-summit.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Join us at the Jenkins Contributor Summit"
+tags:
+- community
+- event
+- jenkins-world
+author: tracymiranda
+---
+
+The Jenkins Contributor summit is where the current and future contributors of the Jenkins project get together.
+This summit will be on Monday, September 17th 2018 in San Francisco, just before link:https://www.cloudbees.com/devops-world/san-francisco[DevOpsWorld | Jenkins World].
+The summit brings together community members to learn, meet and help shape the future of Jenkins. 
+In the Jenkins commmunity we value all types and sizes of contributions and love to welcome new participants.
+
+== Topics
+There are plenty of exciting developments happening in the Jenkins community. 
+The summit will feature a 'State of the Project' update including updates from the Jenkins officers. 
+We will also have updates on the 'Big 5' projects in active development:
+* Jenkins Essentials
+* Jenkins X
+* Configuration as Code
+* Jenkins Pipeline
+* Cloud Native Jenkins Architecture
+
+Plus we will also have updates from our Google Summer of Code students, and more!

```

Comment:
Maybe we could just add it to project updates or to BoF.
Also, there will be 1 student at Max, but we will likely have several mentors. Better to reword at least

Reply:
Have tried to modify - let me know if is better